include/zephyr/drivers/emul_fuel_gauge.h

```diff
@@ -34,6 +34,7 @@ extern "C" {
  */
 __subsystem struct fuel_gauge_emul_driver_api {
 	int (*set_battery_charging)(const struct emul *emul, uint32_t uV, int uA);
+	int (*is_battery_cutoff)(const struct emul *emul, bool *cutoff);
```

Comment:
Do these not get comments?

Reply:
Typically no. The client does not typically interact with the API struct directly but through functions defined in the same header.